src/contacts/ContactsApp.java

```diff
@@ -0,0 +1,56 @@
+package contacts;
+
+import contacts.dao.IRecordDao;
+import contacts.dao.impl.FileRecordDao;
+import contacts.processors.*;
+import contacts.processors.impl.add.AddNewOrganizationProcessor;
+import contacts.processors.impl.add.AddNewPersonProcessor;
+import contacts.processors.impl.phonebook.AddNewRecordProcessor;
+import contacts.processors.impl.add.AddRecordProcessorFactory;
+import contacts.processors.impl.edit.*;
+import contacts.processors.impl.phonebook.*;
```

Comment:
Please set up your IDE no to insert such * in imports

Reply:
Done